agent.go

```diff
@@ -733,11 +733,16 @@ func (s *sandbox) listenToUdevEvents() {
 
 			// Notify watchers that are interested in the udev event.
 			// Close the channel after watcher has been notified.
-			for devPCIAddress, ch := range s.deviceWatchers {
-				if ch != nil && strings.HasPrefix(uEv.DevPath, filepath.Join(rootBusPath, devPCIAddress)) {
+			for devAddress, ch := range s.deviceWatchers {
+				if ch != nil &&
```

Comment:
please remove this condition and create a new one to make this a little bit more readable:

```go
    if ch == nil {
        continue
    }

    if strings.HasPrefix(uEv.DevPath, filepath.Join(rootBusPath, devAddress)) ||
        (strings.Contains(uEv.DevPath, devAddress) && (strings.HasSuffix(devAddress, scsiBlockSuffix) || devAddress == channelSubSystem)) {
```

Reply:
Changed